src/content/api/_endpoints/bank-accounts-archive.js

```diff
@@ -4,7 +4,6 @@ export default {
   request: {
     headers: {
       'X-Api-Key': '<TOKEN>',
-      'Content-Type': 'application/json',
     }
```

Comment:
@seliwe @tom-blake
I've gone through our APIs to ensure 'Content-Type': 'application/json' is included in relevant Post APIs.
The only thing that I noticed is this `Archive Bank Account` API.  It does not have Request body but it has 'Content-Type': 'application/json' included in the curl, so I've removed it.